tests/zenko_e2e/backbeat/package.json

```diff
@@ -16,6 +16,11 @@
     "mocha": "2.3.4",
     "request": "^2.87.0"
   },
+  "devDependencies": {
+    "eslint": "2.13.1",
```

Comment:
Maybe adding eslint should be another PR round, since it's not being used here.

Reply:
sounds good, I added it because my atom editor was complaining and I couldn't figure out an alternative lol